fluent-bit/README.md

```diff
@@ -16,7 +16,9 @@ The `out-sequentialhttp` is a drop-in replacement that sends out records sequent
 
 ### Code modifications
 
-The code of `out-sequentialhttp` is almost identical to `out-http`. The only difference is how msgpack to JSON transcoding/sending to the HTTP backend is done in the `cb_http_flush` function.  
+The code of `out-sequentialhttp` is almost identical to `out-http`. The only difference is how msgpack to JSON transcoding/sending to the HTTP backend is done in the `cb_http_flush` function.
+
+>Note Even though the `out-sequentialhttp` plugin sends out logs one-by-one, retry is still done for the whole batch. If one HTTP request fails, then all of the logs in the respective batch will be resent.
```

Comment:
```suggestion
> **NOTE:** Even though the `out-sequentialhttp` plugin sends out logs one-by-one, retry is still done for the whole batch. If one HTTP request fails, then all of the logs in the respective batch will be re-sent.
```

- Have a look at the [guidelines for the NOTE/CAUTION/TIP panels](https://github.com/kyma-project/community/blob/master/guidelines/content-guidelines/04-formatting.md#panels)
- `re-sent` is spelled with a hyphen as [`resent`](https://dictionary.cambridge.org/dictionary/english/resent) is another word meaning "dislike" or "feel angry".

Reply:
I see, thanks for correcting it ;)